deploy/kubernetes/base/controller/cluster_setup.yaml

```diff
@@ -139,6 +146,9 @@ rules:
   - apiGroups: [""]
     resources: ["events"]
     verbs: ["list", "watch", "create", "update", "patch"]
+  - apiGroups: [""]
```

Comment:
we can be consistent in the comments about why a specific resource verbs is added (i.e here if handle-volume-inuse-error=true, the pod specific rbac is needed). For e.g as you have explained volumeattachments for provisioner, we can put a small comment here. Or we can remove from both the places.

Reply:
Done

Reply:
Is this a resizer flag? Could you mentioned that in the comment here?